crates/iota-node/src/lib.rs

```diff
@@ -1446,7 +1461,10 @@ impl IotaNode {
         state: Arc<AuthorityState>,
         consensus_adapter: Arc<ConsensusAdapter>,
         prometheus_registry: &Registry,
-    ) -> Result<tokio::task::JoinHandle<Result<()>>> {
+    ) -> Result<(
+        tokio::task::JoinHandle<Result<()>>,
+        tokio::sync::oneshot::Sender<()>,
+    )> {
```

Comment:
Would it make sense to make the return type a struct ? so both return values can be named ? (just for clarity & knowing what they are, wouldn't change functionality)